src/util/bundleUtils.js

```diff
@@ -180,10 +184,10 @@ async function getAllDependentLibraries(lib) {
     .filter(
       ra =>
         ra.type === 'depends-on' &&
-        ra.resource.includes('Library') &&
+        (ra.resource?.includes('Library') || ra.url?.includes('Library')) &&
```

Comment:
Also wrong! This resource should be used.

Reply:
Removed the check on `url` and switched to using a `?` to make sure `resource` exists.